settings.toml

```diff
@@ -0,0 +1,4 @@
+# TODO: Do we want to include the settings.toml in the release?
```

Comment:
the file is not included in the output package anyway.
do we need it in the final distribution?
is the user should add this file locally?

Reply:
Thanks. Right now this file is not needed. Should a user encounter problems with the new HDF5 feature, we can instruct them to create this file and put an entry in it that turns the feature toggle off. Shall I remove it from source control entirely for now?

Reply:
Yes, remove it for now

Reply:
Done